compiler-rt/lib/scudo/standalone/wrappers_c.inc

```diff
@@ -247,6 +247,11 @@ void SCUDO_PREFIX(malloc_postinit)() {
 INTERFACE WEAK int SCUDO_PREFIX(mallopt)(int param, int value) {
   if (param == M_DECAY_TIME) {
     if (SCUDO_ANDROID) {
+      // Before changing the interval, reset the memory usage status by doing a
+      // M_PURGE call so that we can minimize the impact of any unreleased pages
+      // introduced by interval transition.
+      SCUDO_ALLOCATOR.releaseToOS(scudo::ReleaseToOS::Force);
```

Comment:
Should this be ForceAll? I presume no, but I wanted to ask.

Reply:
Based on the use case, They show similar memory footprint reduction but have noticeable latency difference. As a result, I think Force is good enough so far.